app/services/database.py

```diff
@@ -16,28 +21,41 @@ def __init__(self, table_name=None):
 
     def load_data(self, table_name=None):
 
-        if table_name is None or len(table_name) is 0:
-            table_name = DATA_PATH
-
-        with open(table_name, "r") as f:
-            return json.load(f)["tmp_uas"]
-
-    def filter_users_with_inclusion_list(self, inclusion_list, limit, db_table=None):
+        gcp_data = gcp_bucket.read_table(table_name)
+        if gcp_data:
+            return gcp_data.get("tmp_uas")
```

Comment:
depending on how the query is ran and stored 'tmp_uas' may not be a valid key

Reply:
'tmp_uas' was a key chosen by Bettina that as far as I know was never supposed to stick. I think we can remove the key altogether when we put in a new database and standardize. But as we currently only have one json file that is structured in this way I suggest we leave it as it is.